pkg/userdata/v1alpha1/cluster_config.go

```diff
@@ -55,6 +55,8 @@ type SchedulerConfig struct {
 // EtcdConfig represents etcd config vals
 type EtcdConfig struct {
 	Image string `yaml:"image,omitempty"`
+	Crt   string `yaml:"crt"`
```

Comment:
It is unclear that this is the CA cert, not the peer, service, or client certs (assuming it _is_ the CA cert, of course).